MediaBrowser.Api/UserService.cs

```diff
@@ -243,7 +243,8 @@ public class UserService : BaseApiService
         /// The user manager.
         /// </summary>
         private readonly IUserManager _userManager;
```

Comment:
Again, probably just delete the XML doc here and remove the whitespace again

Reply:
done

Reply:
The documentation for `UserService` should still remain (even though it's useless 🙄). The StyleCop rules require documentation for classes, but not for private members.